src/components/pageNav/index.jsx

```diff
@@ -1,8 +1,7 @@
 /* eslint-disable react/jsx-handler-names */
 import React from 'reactn';
 import PropTypes from 'prop-types';
-import { Link } from 'gatsby';
-import { Trans, withTranslation } from 'gatsby-plugin-react-i18next';
+import { Trans, withTranslation, Link } from 'gatsby-plugin-react-i18next';
```

Comment:
all `Link` components are now loaded from the i18next plugin so that they redirect to the appropriate locale path